package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gulp-nunjucks",
   "version": "2.0.0",
-  "description": "Precompile Nunjucks templates",
+  "description": "Precompile/render Nunjucks templates",
```

Comment:
Currently you use both `render` and `compile` interchangeably to describe it for the API and docs. Can you pick one?